### DISCUSSION TITLE
How to generate screenshot when running Roborazzi screenshot test locally

### DISCUSSION THREAD

**Discussion by juliewww** [android/nowinandroid#944] [Q&A] [answered]
How to generate screenshots when running Roborazzi screenshot test locally? So that I can update the screenshot with the latest change. Now I only know it can generate the screenshots when running CI.

**Comment by SimonMarquis** [comment, marked as answer]
To generate the screenshots unsing Roborazzi, you'll need to execute this task: `recordRoborazziDemoDebug`

https://github.com/android/nowinandroid/blob/151f877bbe6546c78f8f03f16e56564e285fb508/.github/workflows/Build.yaml#L92-L96
